cmake/dts.cmake

```diff
@@ -108,6 +109,7 @@ if(CONFIG_HAS_DTS)
     ${DTS_FIXUPS}
     --keyvalue ${GENERATED_DTS_BOARD_CONF}
     --include ${GENERATED_DTS_BOARD_H}
+    --structs ${GENERATED_DTS_STRUCTS}
```

Comment:
Is it possible to create generated_dts_board.h from codegen?

Reply:
Yes and No !-). If @erwango finishes his work on a the DTS database that is input to codegen, then codegen can use this input to generate the generated_dts_board.h file. Currently codegen takes it´s input from the generated_dts_board.h.